README.md

```diff
@@ -289,9 +350,9 @@ Sample response:
 }
 ```
 
-#### Get By Id
+#### GetById
  
-In order to retrieve data from your vault using SkyflowIDs, use the **getById(records map[string]interface{})** method. The `records` parameter takes a map that should contain an array of SkyflowIDs to be fetched, as shown below:
+In order to retrieve data from your vault using SkyflowIDs, use the **GetById(records map[string]interface{})** method. The `records` parameter takes a map that should contain an array of SkyflowIDs to be fetched, as shown below:
```

Comment:
I would change, The `records` parameter takes a map that should contain an array of SkyflowIDs to be fetched, as shown below: 

to

The `records` parameter takes a map that has an array of SkyflowIDs to return, as shown below: